workspace/notebook/casework_nsip_examination_timetable_dim_legacy.json

```diff
@@ -107,6 +107,7 @@
 					"    T1.location                     AS Location,\r\n",
 					"    null                            AS EventLineItems,\r\n",
 					"    \"0\"                             AS Migrated,\r\n",
+					"    NULL                            AS Published,\r\n",
```

Comment:
Shouldn't this be T1.Published since we already have the data in standardised table?

Reply:
One request, can we use descriptive names as aliases instead of T1. or a. please? It makes the code a lot easier to read as you don't have the cognitive overhead of remembering what the aliases represent.

Reply:
While I do agree that in some case aliases might be better, in this particular case we are calling views and tables with the same/similar name and aliases will make things a bit more tricky. 
Also T1,T2,T3...etc was used across all harmonised tables when we initially build them, so changing them now would be quite a hassle.

Reply:
We can address it going forwards as we touch code rather than retro fitting throughout. For SQL developers like myself it's poor practice to use aliases like this, we should be using something more obvious instead (e.g. T1 could be exam_timetable, etc).

Reply:
Also the current join order is going to result in a poor execution plan due to the the join order. We should join Horizon timetable to casework first and then use a sub-select to inner join the subset of sourcesystem_fact we're interested in. That way we get the most optimum execution plan from the execution engine. 

The current join order will result in a busy plan and add overhead to the processing time.